packages/rn-tester/Podfile.lock

```diff
@@ -14,7 +14,7 @@ PODS:
     - Flipper-Folly (~> 2.6)
     - Flipper-RSocket (~> 1.4)
   - Flipper-Boost-iOSX (1.76.0.1.11)
-  - Flipper-DoubleConversion (3.2.0)
+  - Flipper-DoubleConversion (3.2.0.1)
```

Comment:
could you remove changes to the `Podfile.lock`, please? They are not related to your PR.

Reply:
@cipolleschi I have done the required changes. Can you please review this?